src/test/java/microsoft/exchange/webservices/data/core/PropertyBagTest.java

```diff
@@ -44,9 +46,21 @@
    */
   @Test(expected=ArgumentException.class)
   public void tryGetPropertyType() throws Exception{
+    PropertyBag pb = createPropertyBag();
+    pb.tryGetPropertyType(String.class, new RecurrencePropertyDefinition("test", "none", null, ExchangeVersion.Exchange2010_SP2), new OutParam<String>());
+  }
```

Comment:
Is there a chance you will also add maybe only some tests for PropertyDefinitions?